extensions/wikia/GlobalNavigation/templates/GlobalNavigationAccountNavigation_index.php

```diff
@@ -1,24 +1,50 @@
-<ul id="AccountNavigation" class="AccountNavigation table-cell">
-	<li class="account-navigation-first-item">
-		<div class="links-container<?php if ($isAnon): ?> anon <?php endif; ?>">
-			<?= $navItemLinkOpeningTag ?>
+<ul id="AccountNavigation" class="AccountNavigation table-cell <?= $enableNewAuth ? 'newAuth' : '' ?> ">
+	<?php if ($enableNewAuth): ?>
+		<li class="account-navigation-item">
+			<div class="links-container<?php if ($isAnon): ?> anon <?php endif; ?>">
+				<?= $navItemLinkOpeningTag ?>
 				<div class="avatar-container<?= $avatarContainerAdditionalClass ?>">
 					<?= $profileAvatar ?>
 				</div>
-			</a>
-			<?php if ($isAnon) : ?>
-				<span class="sign-in-label"><?= wfMessage( 'global-navigation-sign-in' )->escaped(); ?></span>
-			<? endif; ?>
-			<img class="chevron" src="data:image/gif;base64,R0lGODlhAQABAIABAAAAAP///yH5BAEAAAEALAAAAAABAAEAQAICTAEAOw%3D%3D"/>
-		</div>
-		<?php if ( !$isAnon ): ?>
-		<ul class="user-menu subnav global-nav-dropdown">
-			<?php foreach( $userDropdown as $link ): ?>
-				<li><?= $link ?></li>
-			<?php endforeach; ?>
-		</ul>
-		<?php else:
-			echo $loginDropdown;
-		endif; ?>
-	</li>
+				</a>
+				<?php if ($isAnon) : ?>
+					<span class="auth-label register"><?= wfMessage( 'global-navigation-register' )->escaped(); ?></span>
+					<span class="auth-label"><?= wfMessage( 'global-navigation-or' )->escaped(); ?></span>
+					<span class="auth-label sign-in"><?= wfMessage( 'global-navigation-sign-in' )->escaped(); ?></span>
```

Comment:
we need to prepare proper changes for that file when we will have some feedback from the designers about login & reg buttons behaviour